src/components/tooltip/tooltip.test.tsx

```diff
@@ -174,41 +163,44 @@ describe("Tooltip", () => {
         large: isTopOrBottom(position) ? 10 : 12,
       });
 
-      describe.each(positions)("and position is %s", (position) => {
-        it.each<InputSizes>(["small", "medium", "large"])(
-          "sets the offset as expected when size is %s",
-          (size) => {
-            const useFloatingSpy = jest.spyOn(floatingUi, "useFloating");
-            render({
-              isVisible: true,
-              position,
-              isPartOfInput: true,
-              inputSize: size,
-            });
-
-            let middleware;
-            if (useFloatingSpy.mock.calls[0][0]?.middleware?.[0]) {
-              middleware = useFloatingSpy.mock.calls[0][0]?.middleware?.[0];
+      describe.each(["top", "bottom", "left", "right"] as const)(
```

Comment:
comment (non-blocking): this is the prime example of a test that we need to refactor 😆. Again happy for it to be part of future work